interface/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:12.18.2-alpine
+
+COPY . /code
+
+WORKDIR /code
+
+ENV PATH /code/node_modules/.bin:$PATH
```

Comment:
It might be better to install the node_modules needed here, instead of relying on a developer having them in their current working folder.

Reply:
It just adds it to the node js PATH, and installs the node modules in the directory itself. You can have a look here
https://github.com/PecanProject/BETYdb-YABA/blob/676e63deffe05d90fd65c49f4294c4fddf11b183/interface/Dockerfile#L9-L11